psalm.baseline.xml

```diff
@@ -8,7 +8,8 @@
       <code>$this-&gt;cssSelectorConverter === null</code>
       <code>$this-&gt;matchingUninlinableCssRules === null</code>
     </DocblockTypeContradiction>
-    <InvalidArgument occurrences="1">
+    <InvalidArgument occurrences="3">
+      <code>$cssRule</code>
       <code>$cssRule</code>
     </InvalidArgument>
```

Comment:
`occurrences="3"` but there are only 2 in the list - ?

Reply:
Yes, that's strange indeed. Still, this is what Psalm generates (for whatever reason), and I'd like to keep it so we don't need to always change this manually when re-generating the file.

Reply:
No, this doesn'tt change anything.

Reply:
OK.